### DISCUSSION TITLE
如何在应用部署界面，填写dockerconfigjson的值？

### DISCUSSION THREAD

**Discussion by CyrusLiu158** [labring/sealos#3407] [Q&A]
如图所示，私有Docker的时候，在哪里填写呀？
<img width="841" alt="image" src="https://github.com/labring/sealos/assets/3727646/acee4a2e-4f15-4ebb-b2d0-bb53860a6334">

**Comment by zzjin** [comment]
![image](https://github.com/labring/sealos/assets/679757/c59d648e-d59a-4ccd-a65c-90e33e45ec36)
